megalinter/Linter.py

```diff
@@ -140,6 +140,11 @@ def __init__(self, params=None, linter_config=None):
             self.name = (
                 self.descriptor_id + "_" + self.linter_name.upper().replace("-", "_")
             )
+
+        # Override default executable
+        self.cli_executable = config.get(
```

Comment:
Add if config.exists(self.name + "_CLI_EXECUTABLE")  before setting it maybe ? :)

Reply:
Because cli_executable is sometimes already defined in descriptors, so when you do that, you always replaces the value from descriptor with None :)

Reply:
+ you need to update documentation (build.py) and json schema, to add new variables :)

Reply:
> 
> 
> Add if config.exists(self.name + "_CLI_EXECUTABLE") before setting it maybe ? :)

Fixes by commit ffd191d

Reply:
>     * you need to update documentation (build.py) and json schema, to add new variables :)

Fixes by commits 8b64908, c464cf6 and df3055f